### DISCUSSION TITLE
Mensagens no console

### DISCUSSION THREAD

**Discussion by lfcalabria** [balta-io/forum#314] [Tira Dúvidas] [answered]
Estou fazendo a trilha de desenvolvedor flutter, estou no curso do app contacts, no módulo da camera, neste modulo devemos fazer um print para ver se está ok o take-photo, mas não consigo acompanhar este print porque tem muitas mensagens chegando. Como faço para desligar este monte de mensagens?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @lfcalabria, como vai?
Estas mensagens são logs de debug e elas incluem o print.

O que você pode fazer é filtrar as mensagens, apertando `Control + F` no terminal e filtrando por `"I/Flutter"`
